SPARQLWrapper/KeyCaseInsensitiveDict.py

```diff
@@ -2,47 +2,46 @@
 
 """
 A simple implementation of a key case-insensitive dictionary.
-
 ..
   Developers involved:
-
   * Ivan Herman <http://www.ivan-herman.net>
   * Sergio Fernández <http://www.wikier.org>
   * Carlos Tejo Alonso <http://www.dayures.net>
   * Alexey Zakhlestin <https://indeyets.ru/>
-
   Organizations involved:
-
   * `World Wide Web Consortium <http://www.w3.org>`_
   * `Foundation CTIC <http://www.fundacionctic.org/>`_
-
   :license: `W3C® Software notice and license <http://www.w3.org/Consortium/Legal/copyright-software>`_
 """
 
+from typing import Dict, TypeVar
+
+K = str
```

Comment:
Same for this, rather prefix it with an underscore, but I would say it is best to not alias str and just use it directly.

Reply:
7dc98eb